src/layout/defaultContainer.js

```diff
@@ -1,37 +1,62 @@
+import { useSelector, useDispatch } from "react-redux"
+import { useNavigate } from "react-router-dom"
+import { logOut, reset } from "../features/auth/authSlice"
 import Nav from "../elements/Nav"
 
 const dummyUser = {
   name: "Halyna Pravdych",
   position: "frontend dev",
-};
+}
 const dummyNav = [
   "all projects",
   "new project",
   "new skill",
   "update cv",
   "messages",
   "settings",
-];
+]
 
 function DefaultContainer({ authorized, ...props }) {
+  const navigate = useNavigate()
+  const dispatch = useDispatch()
+  const { user } = useSelector((state) => state.auth)
+
+  const onLogOut = () => {
+    dispatch(reset())
+    dispatch(logOut())
+    navigate("/auth/sign-in")
+  }
```

Comment:
Try to wrap any created function inside component into `useCallback` so it won't be recreated every time during re-render.